integration/nwo/network.go

```diff
@@ -770,18 +744,13 @@ func (n *Network) GenerateConfigTree() {
 	}
 }
 
-// Bootstrap generates the cryptographic material, orderer system channel
-// genesis block, and create channel transactions needed to run a fabric
-// network.
+// Bootstrap generates the cryptographic material and create channel transactions
+// needed to run a fabric network.
```

Comment:
Bootstrap generates the cryptographic material and application channels genesis blocks,